apps/web/src/components/GasPrice.tsx

```diff
@@ -0,0 +1,33 @@
+import { api } from "~/api-client";
+
+export function LatestGasPrice() {
+  const { data } = api.block.getLatestGasPrice.useQuery();
+
+  if (!data) {
+    return null;
+  }
+
+  return (
+    <div className="flex items-center justify-center gap-2 rounded px-3 py-1">
+      <GasSVG />
+      <div>
+        <span className="font-bold">Blob gas price: </span>
+        {data.blobGasPrice.toString()}
+        <span className="test-sm opacity-80"> wei</span>
+      </div>
+    </div>
+  );
+}
+
+function GasSVG() {
```

Comment:
Let's do the same than described [here](https://github.com/Blobscan/blobscan/pull/464#discussion_r1700386719)